spec/spec_helper.rb

```diff
@@ -16,6 +16,7 @@
 gc_counter = -1
 
 RSpec.configure do |config|
+  config.alias_example_to :they
```

Comment:
I get what this is doing, but seems unnecessary for just one instance of using it?